libc/config/baremetal/arm/headers.txt

```diff
@@ -8,4 +8,5 @@ set(TARGET_PUBLIC_HEADERS
     libc.include.stdlib
     libc.include.string
     libc.include.strings
+    libc.include.sys_queue
```

Comment:
it would be nice to add this header to the other targets as well since it should still work, and that will make sure that it's being tested by our upstream builders.

Reply:
Done.